<h1 style='font-size:40px'> S3 Security</h1>

<h2 style='font-size:30px'> S3 Lifecycle Rules (with S3 Analytics)</h2>
<div> 
    <ul style='font-size:20px'> 
        <li> 
        </li>
    </ul>
</div>

<h3 style='font-size:30px;font-style:italic'> Chaves Criptográficas</h3>
<div> 
    <ul style='font-size:20px'> 
        <li>
            Chaves criptográficas são strings usadas por algoritmos de criptografia para a alteração de dados.
        </li>
        <li>
            De forma bem superficial, essa string servirá como um "de-para" para a conversão dos dados.
        </li>
    </ul>
</div>

In [1]:
! git add .
! git commit -am 'Pesquisar sobre o Server-Side Encryption na AWS e as modalidades de chave suportadas'
! git push

[master 33127e2] Aula 142
 5 files changed, 244 insertions(+), 80 deletions(-)
 create mode 100644 .ipynb_checkpoints/14_s3_security-checkpoint.ipynb
 create mode 100644 14_s3_security.ipynb
 create mode 100644 img/13_dyamo_s3.png
Enumerating objects: 11, done.
Counting objects: 100% (11/11), done.
Delta compression using up to 24 threads
Compressing objects: 100% (7/7), done.
Writing objects: 100% (7/7), 27.51 KiB | 27.51 MiB/s, done.
Total 7 (delta 5), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (5/5), completed with 4 local objects.
To https://github.com/felipesveiga/aws-developer.git
   477162f..33127e2  master -> master


<p style='color:red'> Defini encyption keys; Pesquisar sobre o Server-Side Encryption na AWS e as modalidades de chave suportadas; Aula 142</p>